src/muparserx.mk

```diff
@@ -0,0 +1,28 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := muparserx
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 3.0.5
+$(PKG)_CHECKSUM := d8f646aac4099973b70ae84e1c7cb411abb6fe0b
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $($(PKG)_SUBDIR).tar.gz
+$(PKG)_URL      := http://sourceforge.net/projects/mxedeps/files/$($(PKG)_FILE)
```

Comment:
What was fixed? `http://sourceforge.net/projects/mxedeps/files` is not a valid primary source. Or secondary for that matter if it isn't part of a mirror or cdn.